python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -44,6 +45,7 @@
 PARTIAL_PARSE_FILE_NAME = "partial_parse.msgpack"
 
 
+@public
```

Comment:
Classes don't need public decorators - only methods do. Any class exported at the top-level is considered public.